### ISSUE TITLE
Is it possible to deploy openelb as daemonset?

### ISSUE THREAD

**Issue by lwabish** [openelb/openelb#358]
**Is your feature request related to a problem? Please describe.**

In my case, like edge computing , the node amount of my cluster would probably not exceed 10, so I would like to run openelb-manager as daemonset.
But now the helm chart doesn't support switching deployment to daemonset.

**Describe the solution you'd like**

Offer a switch for users to decide deploying openelb-manager with daemonset or deployment.

**Describe alternatives you've considered**

**Additional context**

**Comment by renyunkang** [comment]
For the version of `openelb` you are using, you may consider increasing the number of replicas of `openelb-manager`. Modify the scheduling rules so that it can be dispatched to different nodes.

**Comment by lwabish** [comment]
> For the version of `openelb` you are using, you may consider increasing the number of replicas of `openelb-manager`. Modify the scheduling rules so that it can be dispatched to different nodes.
Yes, indeed. This is also what I am doing now to achive the goal. 
I am just considering offering a switch in the chart , which is maybe more easy for end users to deploy, without considering node amount, scheduling, lables, etc.
